SD-K9/src/app/pages/home/home.page.html

```diff
@@ -10,11 +10,11 @@
   
       <ion-item>
         <ion-label position="floating">Choose starting point...</ion-label>
-        <ion-input formControlName="source"></ion-input>
+        <ion-input formControlName="source" [value]="startLocation"></ion-input>
       </ion-item>
       <ion-item>
         <ion-label position="floating">Choose destination...</ion-label>
-        <ion-input formControlName="destination"></ion-input>
+        <ion-input formControlName="destination" [value]="endLocation"></ion-input>
       </ion-item>
       <ion-list >
         <ion-segment (ionChange)="mode($event)" value="DRIVING"> 
```

Comment:
Just a heads up, this form doesnt exist in master anymore, its using a state store. Do you want to pull from master and then modify your code?